internal/pkg/service/appproxy/http/app.go

```diff
@@ -53,7 +29,7 @@ func appIDMiddleware(publicURL *url.URL) middleware.Middleware {
 	}
 }
 
-func parseAppID(publicURL *url.URL, host string) (AppID, bool) {
+func parseAppID(publicURL *url.URL, host string) (string, bool) {
```

Comment:
The benefit is that you cannot confuse different string values that have different meanings, because it is protected by the type. 

I see, it is used only in a telemetry attribute.